x/datadeal/types/expected_keepers.go

```diff
@@ -10,6 +11,7 @@ type AccountKeeper interface {
 
 	SetAccount(ctx sdk.Context, acc authtypes.AccountI)
 	GetAccount(ctx sdk.Context, addr sdk.AccAddress) authtypes.AccountI
+	GetPubKey(sdk.Context, sdk.AccAddress) (cryptotypes.PubKey, error)
```

Comment:
I think there is no need to have this func.

Reply:
Yes, you're right.